src/Compilers/CSharp/Test/Syntax/Parsing/RawInterpolatedStringLiteralCompilingTests.cs

```diff
@@ -0,0 +1,1757 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Parsing;
+
+public class RawInterpolatedStringLiteralCompilingTests : CompilingTestBase
+{
+    private static string Render(string markup, string? normalize)
+    {
+        markup = markup.Replace('␠', ' ').Replace('␉', '\t');
```

Comment:
nit: could have used case-sensitive `SPACE` and `TAB`, may offer improved legibility

Reply:
Hrmm.  But 8 spaces would be SPACESPACESPACESPACESPACESPACESPACESPACE

I'M not sure if that would be good :-)

Reply:
The longest sequence is 4 or 5.
Maybe angled brackets then? `<space><space><space><space><space>`

Reply:
TBH, i genuinely think the approach in the PR is good.  it's simple, a single char, and those unicode chars actually are intended for this purpose. (i.e. 'SP' is the [unicode symbol for space](https://unicode-table.com/en/2420/)).

Reply:
going to resolve this.  i love the unicode escapes :D

Reply:
That's fine :-P